otter/convergence/planning.py

```diff
@@ -587,24 +619,54 @@ def remove_node_from_lb(node):
             [(node.description.lb_id, node.cloud_server_id)]))
 
 
-def change_lb_node(node, description):
+def change_lb_node(node, description, lb, now, timeout):
     """
-    Change the configuration of a load balancer node.
+    Change the configuration of a load balancer node to desired description.
+    If CLB has health monitor enabled and the node is DRAINING then it will be
+    ENABLEDed.
 
-    :ivar node: The node to be changed.
+    :param node: The node to be changed.
     :type node: :class:`ILBNode` provider
-
-    :ivar description: The description of the load balancer and how to add
+    :param description: The description of the load balancer and how to add
         the server to it.
     :type description: :class:`ILBDescription` provider
+    :param float now: Current time in EPOCH
+    :param float timeout: How long can node remain OFFLINE after adding
+        in seconds?
+
+    :return: :obj:`IStep` object or None
```

Comment:
Just a thought for the future - would a "do-nothing step" object make more sense than None here?